ui/robolectric/BasicSample/app/build.gradle

```diff
@@ -0,0 +1,32 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 26
+    buildToolsVersion rootProject.buildToolsVersion
+    defaultConfig {
+        applicationId "com.example.android.testing.robolectric.BasicSample"
```

Comment:
I don't think application id's usually have capital letters in do they? Is this the pattern in the other samples?

Reply:
Yes, it matches the other examples (it matches the package name which is uppercase).